workflow/exp/exp.conus12km

```diff
@@ -19,6 +19,10 @@ export ZETA_LEVELS="L60.txt"
 export DA_IODA="true"
 export DA_JEDI="true"
 export OBSPATH="local.setup"
```

Comment:
OBSPATH is defined later in this same file. Do you still need this?

Reply:
Yes, this may be removed in the future.
I keep them right now to facilitate the comparison between the realtime versions and retro versions.